src/main/java/com/mateacademy/sql/ConnectorUtil.java

```diff
@@ -0,0 +1,128 @@
+package com.mateacademy.sql;
+
+import org.apache.log4j.Logger;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.sql.SQLException;
+import java.sql.PreparedStatement;
+import java.util.Properties;
+
+public class ConnectorUtil {
+    private static final Logger LOGGER = Logger.getLogger(ConnectorUtil.class);
+    private Connection connection;
```

Comment:
Why did you make it like a field, but not a local variable per method?

Reply:
Because i use one connection for whole program

Reply:
It is better to open and close connection each time when you want to take or put information into/from DB. But not hang a connection.  Now you have just one user. What will happen if many users use your program at the same time?